src/main/webapp/WEB-INF/tags/shared/feedbackSubmissionEdit/response.tag

```diff
@@ -45,6 +52,34 @@
       </c:choose>:
   </div>
   <c:choose>
+    <c:when test="${questionWithResponses.feedbackParticipantCommentsOnResponsesAllowed}">
+      <button type="button" class="btn btn-default btn-xs icon-button pull-right show-frc-${addOrEdit}-form"
+              id="button_add_comment-${responseRecipientIndex}-${responseGiverIndex}-${questionIndex}"
+              data-recipientindex="${responseRecipientIndex}" data-giverindex="${responseGiverIndex}"
+              data-qnindex="${questionIndex}" data-toggle="tooltip" data-placement="top"
+              title="<%=Const.Tooltips.COMMENT_ADD%>"
+              <c:if test="${data.preview or (not data.submittable)}"> disabled style="background: #66727A;"</c:if>>
+        <span class="glyphicon glyphicon-comment glyphicon-primary"></span>
+      </button>
+      <div class="${divClassType}">
+          ${response.submissionFormHtml}
+        <br>
+        <ul class="list-group" id="responseCommentTable-${responseRecipientIndex}-${responseGiverIndex}-${questionIndex}"
+            style="${not empty response.feedbackParticipantCommentOnResponse ? 'margin-top:15px;': 'display:none'}">
+          <c:if test="${isCommentOnResponse}">
+            <shared:feedbackResponseCommentRow frc="${response.feedbackParticipantCommentOnResponse}"
```

Comment:
Use response id?

Reply:
Sorry, I didn't understand. Use response id for?

Reply:
Sorry, I mean comment id. 

But as we are going to write new tag just for feedback submission, my comment is no longer valid.